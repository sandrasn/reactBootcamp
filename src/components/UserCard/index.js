import React from "react";

const UserCard = ({ image, username }) => (
  <div className="user">
    <img className="user-img" src={image} />
    <p className="username"> {username}</p>
  </div>
);

export default UserCard;
