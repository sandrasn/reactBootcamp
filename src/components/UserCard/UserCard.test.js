import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UserCard from "./index";

describe("Testing UserCard", () => {
  const wrapper = shallow(<UserCard />);

  it("renders properly", () => {
    expect(wrapper.length).toEqual(1);
  }); //it === test

  it("renders a snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
