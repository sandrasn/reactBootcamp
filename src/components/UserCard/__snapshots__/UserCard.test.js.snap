// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UserCard renders a snapshot 1`] = `
<div
  className="user"
>
  <img
    className="user-img"
  />
  <p
    className="username"
  >
     
  </p>
</div>
`;
